pkg/api/v1/types.go

```diff
@@ -111,8 +125,6 @@ const (
 	AgentPoolProfileRoleCompute AgentPoolProfileRole = "compute"
 	// AgentPoolProfileRoleInfra is the infra role.
 	AgentPoolProfileRoleInfra AgentPoolProfileRole = "infra"
-	// AgentPoolProfileRoleMaster is the master role.
-	AgentPoolProfileRoleMaster AgentPoolProfileRole = "master"
```

Comment:
Is this going to be read only field for masters or is it even needed in the public API model that `MasterPoolProfile` is a separate?

Reply:
This constant is removed from the external api model with this line since it is not in the public `MasterPoolProfile`.  It is retained in the internal model (`pkg/types`) for use after conversion.